src/modules/todo/api/index.ts

```diff
@@ -1,7 +1,23 @@
+import { Layout } from 'react-grid-layout'
 import type { TodoItem } from '@/modules/todo/types/todo.types'
+import axios from 'axios'
 
 type PostTodoData = Omit<TodoItem, 'id'>
 
+const LAYOUT_KEY = 'layout'
+
+const saveToLS = (data: Layout[]) => {
+  localStorage.setItem(LAYOUT_KEY, JSON.stringify(data))
+}
+
+const getFromLs = (): Layout[] | undefined => {
```

Comment:
Create service to getting data from the local storage. You can also create generic for this, so you could pass params and get result typing. For example it will be something like this.
`LocalStorageService.get<Layout[]>(LAYOUT_KEY)`

This service shoud be in `modules/core/services`